spec/betterc.dd

```diff
@@ -4,38 +4,88 @@ $(SPEC_S Better C,
 
 $(HEADERNAV_TOC)
 
-`-betterC` is a command-line flag for `dmd`,
-which restricts the compiler's support of certain runtime features.
-Notably, D programs or libraries compiled with `betterC` aren't linked with Druntime.
-The use of compile-time features is not restricted in any way.
+    $(P It is straightforward to link C functions and libraries into D programs.
+    But linking D functions and libraries into C programs is not straightforward.
+    )
 
-Limiting a program or library to this subset of runtime features is useful
-when targeting constrained environments where the use of such features is not practical or possible.
-Additionally, this also makes embedding D libraries in larger projects easier by:
+    $(P D programs generally require:)
 
-$(OL
-    $(LI Simplifying the process of integration at the build-system level)
-    $(LI Removing the need to ensure that Druntime is properly initialized on calls to the library, when an initialization step is not performed before the library is used.)
-    $(LI Mixing memory management strategies (GC + manual memory management) can sometimes be tricky, hence removing D's GC from the equation may be a good solution)
-)
+    $(OL
+    $(LI The D runtime library (Phobos) to be linked in, because many features of
+    the core language require runtime library support.)
+    $(LI The `main()` function to be written in D, to ensure that the required
+    runtime library support is properly initialized.)
+    )
```

Comment:
A general remark: I always use indentation for OL/OL (and used it here), because as you have recently discovered yourself, it's quite tricky to keep track of all parentheses in Ddoc. Same motivation applies to indent the content in `$(P`.